extensions/wrapper/wrapper-api/src/main/java/de/sovity/edc/ext/wrapper/api/usecase/UseCaseResource.java

```diff
@@ -59,4 +61,11 @@ List<UiDataOffer> queryCatalog(
         @Valid @NotNull
         CatalogQuery catalogQuery
     );
+
+    @POST
+    @Path("policy-definition")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(description = "Create a new Policy Definition")
+    IdResponseDto createPolicyDefinitionUseCase(PolicyCreateRequest policyCreateRequest);
```

Comment:
The UseCaseApi needs to be accessed separately. The method name will be passed into the API clients as method name, thus the usage would be

`edcClient.useCaseApi().createPolicyDefinitionUseCase(...)`, I believe we can drop the "UseCase" from the method name.

Reply:
I implemented it like this in the first place, but it seems if there are the same names (also located in different APIs) the `operationId` in the OpenApi spec will be generated to createPolicyDefinition**1**.

`createPolicyDefinition` is also used in the UiApi.